MiniAn/minian_parameters.py

```diff
@@ -152,7 +152,12 @@ def __init__(self, danse_parameters):
         self.update_all_func_params()
         self.parameters[defs.Parameters.danse.ADVANCED_SETTINGS] = self.advanced_settings.copy()
 
-
+        self.params_crossRegister = {
+            'fname_crossReg': self.paths[mn_defs.DictionaryKeys.CrossReg.FNAME_CROSSREG],
+            'h5path_images': self.paths[mn_defs.DictionaryKeys.CrossReg.H5PATH_IMAGES_CROSSREG],
+            'h5path_roi': self.paths[mn_defs.DictionaryKeys.CrossReg.H5PATH_ROI_CROSSREG],
+            'crossReg': self.parameters[mn_defs.DictionaryKeys.CrossReg.CROSS_REGISTRATION] 
```

Comment:
I think you don't need to have the `'crossReg':` key and can directly use `minian_parameters.parameters[mn_defs.DictionaryKeys.CrossReg.CROSS_REGISTRATION]` like for motion correction in the `correction_motion(...)` function